src/draw/bar/widgets.rs

```diff
@@ -63,6 +63,18 @@ impl Text {
         }
     }
 
+    /// Changes the color of the text.
+    pub fn set_fg(&mut self, color: Color) {
+        self.require_draw |= color != self.fg;
```

Comment:
Sorry if this is just me being dim, but are these `|=` expressions intended to set `self.require_draw` if the color is actually changing? If so, I'd prefer something more readable like:
```rust
if color != self.fg {
    self.fg = color;
    self.require_draw = true;
}
```

Reply:
The problem there is that it triggers unnecessary draws in cases where nothing changes.  How do you feel about:

    let changing = color != self.fg;
    self.require_draw |= changing;

or 

    let changing = color != self.fg;
    self.require_draw = self.require_draw || changing;

Reply:
np, I'll make that change